### ISSUE TITLE
Importing nevergrad issues  InefficientSettingsWarning warning

### ISSUE THREAD

**Issue by jieru-hu** [facebookresearch/nevergrad#1208]
## Steps to reproduce
```
>>> import nevergrad as ng
/Users/jieru/opt/anaconda3/envs/pytest38/lib/python3.8/site-packages/nevergrad/optimization/differentialevolution.py:106: InefficientSettingsWarning: DE algorithms are inefficient with budget < 60
  warnings.warn(
>>> print(ng.__version__)
0.4.3.post7

# downgrad nevergrad, no warnings
>>> import nevergrad as ng
>>> print(ng.__version__)
0.4.3.post4

```

## Observed Results

  * warnings issued when importing

## Expected Results

  * no warning :)

**Comment by jrapin** [comment]
Fixed in #1223

**Comment by jrapin** [comment]
@jieru-hu this is now fixed, with a new test to make sure there is no more warnings at import. I pushed an updated version 0.4.3.post8 on pypi
